README.md

```diff
@@ -38,6 +38,12 @@ Please follow this [repo](https://github.com/wesbos/eslint-config-wesbos/) by We
 ### Install dependencies
 
 ```
+### Refer this https://www.npmjs.com/package/alexa-skill-local for usage #####
+
 npm install
 npm install -g alexa-skill-local
-```
\ No newline at end of file
+```
+
+Install ask dependency
+npm install --save ask-sdk
```

Comment:
This is not needed since the dependency is already in there the package.json.

Reply:
Okay got it.
changes already added in package.json